node/src/service.rs

```diff
@@ -307,22 +314,26 @@ where
 		);
 		let spawner = task_manager.spawn_handle();
 
-		let polkadot_backend = polkadot_full_node.backend.clone();
-
-		let params = StartCollatorParams {
+		let parachain_consensus = build_relay_chain_consensus(BuildRelayChainConsensusParams {
 			para_id: id,
-			block_import,
```

Comment:
This is wrong. We need the frontier block import here. This is different than the cumulus template where they do not use Frontier.